datumaro/plugins/vgg_face2_format.py

```diff
@@ -43,30 +62,46 @@ def _load_items(self, path):
 
         for row in landmarks_table:
             item_id = row['NAME_ID']
-            image_path = osp.join(self._dataset_dir, self._subset,
-                item_id + VggFace2Path.IMAGE_EXT)
-            annotations = []
+            label = None
+            if '/' in item_id:
+                label_name = item_id.split('/')[0]
+                label = self._categories[AnnotationType.label].find(label_name)[0]
+                if label is not None:
+                    item_id = item_id[len(label_name) + 1:]
```

Comment:
Why this can happen?

Reply:
If for example item.id = a/1 and there is no label

Reply:
Maybe add a folder for unlabeled items (like in ImageNet)?

Reply:
I think it is a good idea, because otherwise we require `labels.txt`.